assembly/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <artifactId>infra-provider-kafka</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.homeaway.streamplatform</groupId>
+                <artifactId>infra-provider</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
This was necessary due to migrating the StreamCreationException to infra-provider for the InfraManager (due to it originally being in KafkaManager). The other classes in the core module that use that exception ran into compilation problems even though the infra-provider module is in the core module's pom.

Reply:
of course though if you know of a different (read: better) way of accomplishing the same definitely let me know and I'll make that change :)

Reply:
After our discussion... wondering if we can just move `InfraManager` in core and delete `infra-provider` all together?

Reply:
Commenting here to document a discussion that Rene and I had discussing this pom change - in order to avoid code smell, we will be migrating the InfraManager back into the core module and delete the infra-provider module. This will remove the various dependencies in the core/assembly pom.xml files. This also makes more sense from an architectural standpoint given that infra-provider-kafka is for specific implementations whereas InfraManager is an interface facilitating that.